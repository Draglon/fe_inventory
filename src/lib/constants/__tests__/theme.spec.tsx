import * as theme from "../theme";

it("theme constants match snapshot", () => {
  expect(theme).toMatchSnapshot();
});
