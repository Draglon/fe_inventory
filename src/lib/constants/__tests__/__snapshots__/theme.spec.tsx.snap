// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`theme constants match snapshot 1`] = `
{
  "THEME": {
    "dark": "dark",
    "light": "light",
  },
}
`;
